tests/graphql_lists/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,19 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`lists.graphql 1`] = `
+{
+  shortWithList(list: [1, 2, 3])
+  longWithList(list: ["hello world this is a very long string!","hello world this is a very long string!", "hello world this is a very long string!"])
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+{
+  shortWithList(list: [1, 2, 3])
+  longWithList(
+    list: [
+      "hello world this is a very long string!",
+      "hello world this is a very long string!",
+      "hello world this is a very long string!"
```

Comment:
Yeah, if you could use the trailingCommas=es5 to decide when to print them it would be great